controllers/setup.go

```diff
@@ -99,7 +99,10 @@ func setupManager(ctx context.Context, cancel context.CancelFunc, mgr controller
 		)
 	}
 
-	var requiredCrds []string
+	requiredCrds := []string{
+		"virtualMachine.kubevirt.io",
```

Comment:
Can you put this into the vm_controller instead of defining it just here? e.g. adding a `WatchTypes` func to vm_controller?

Reply:
@0xFelix Isn't that function related only to operands? Is `vm_controller` considered as an operand?

Reply:
My point is that this is a requirement of `vm_controller.go` but when defined like this in `setup.go` it is not obvious. The suggestion was to add something similar to a `WatchTypes()` func to `vm_controller.go`.

Reply:
+1 for Felix's idea. It will make it clear that `virtualMachine.kubevirt.io` is needed by the `vm_controller`.

If the CRD is missing, the controller should not be added to the manager. You can add an `if` condition round lines  143-150